### DISCUSSION TITLE
Tip for closing tabs

### DISCUSSION THREAD

**Discussion by MestrePadawan** [zen-browser/desktop#1850] [Ideas]
Hello, thank you for the project. I have a suggestion: put a button to close a specific open tab. It is inconvenient to close a tab using the keyboard or by right-clicking and selecting "Close Tab".

Thank you

**Comment by Axenide** [comment]
I used to have a userChrome.css that replaced the favicon for the close button when hovered. Maybe this would be a good solution, at least when the tab bar is collapsed.

Meanwhile, you can close tabs with middle-click. :)

**Comment by GitFOSSinstead** [comment]
I believe if you hover over the tab and use the middle mouse button it'll close the tab too.

**Comment by MestrePadawan** [comment]
thanks for the replies guys, but there is no middle click on notebooks. :(

**Reply by Axenide** [reply]
Oh shoot, well you can middle-click with three fingers (if supported) but it is kind of a hassle lol.
You can always <kbd>Ctrl</kbd>+<kbd>W</kbd>.
